packages/react/src/components/builder-block.component.tsx

```diff
@@ -450,7 +451,7 @@ export class BuilderBlock extends React.Component<
   getElement(index = 0, state = this.privateState.state): React.ReactNode {
     const { child, fieldName } = this.props
     const block = this.block
-    let TagName = (block.tagName || 'div').toLowerCase()
+    let TagName: string | typeof Link = (block.tagName || 'div').toLowerCase()
```

Comment:
Is it a common pattern to use `typeof` for getting the 'type' of React components?

Reply:
Not sure, but it's less verbose than writing React.SFC<React.AnchorHTMLAttributes<HTMLAnchorElement>>